src/domains/receipts-app/terraform.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
```

Comment:
replace it with the following [one](https://github.com/pagopa/pagopa-infra/blob/main/src/core/terraform.sh)
extended with https://github.com/dineshba/tf-summarize